bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -650,4 +650,530 @@ public Point closest(Point p) {
+        { 6110, 0.320061093999071, 0.329870482936352 },
+        { 6120, 0.319881469970083, 0.329702147726276 },
+        { 6130, 0.319702517855622, 0.329534261213777 },
+        { 6140, 0.319524234349642, 0.329366822201013 },
+        { 6150, 0.319346616165388, 0.329199829489774 },
+        { 6160, 0.319169660035263, 0.32903328188155 },
+        { 6170, 0.318993362710712, 0.328867178177603 },
+        { 6180, 0.3188177209621, 0.328701517179031 },
+        { 6190, 0.318642731578585, 0.32853629768684 },
+        { 6200, 0.318468391368004, 0.328371518502004 },
+        { 6210, 0.318294697156752, 0.328207178425535 },
+        { 6220, 0.31812164578966, 0.328043276258542 },
+        { 6230, 0.31794923412988, 0.327879810802294 },
+        { 6240, 0.317777459058767, 0.327716780858286 },
+        { 6250, 0.317606317475763, 0.32755418522829 },
+        { 6260, 0.317435806298278, 0.32739202271442 },
+        { 6270, 0.317265922461579, 0.327230292119189 },
+        { 6280, 0.317096662918675, 0.327068992245565 },
+        { 6290, 0.316928024640201, 0.326908121897025 },
+        { 6300, 0.316760004614306, 0.326747679877612 },
+        { 6310, 0.316592599846543, 0.326587664991987 },
+        { 6320, 0.316425807359756, 0.326428076045483 },
+        { 6330, 0.316259624193968, 0.326268911844154 },
+        { 6340, 0.316094047406274, 0.326110171194829 },
+        { 6350, 0.315929074070729, 0.325951852905159 },
+        { 6360, 0.315764701278241, 0.325793955783666 },
+        { 6370, 0.315600926136464, 0.325636478639792 },
+        { 6380, 0.315437745769688, 0.325479420283945 },
+        { 6390, 0.315275157318735, 0.325322779527545 },
+        { 6400, 0.315113157940854, 0.325166555183068 },
+        { 6410, 0.314951744809611, 0.325010746064094 },
+        { 6420, 0.314790915114793, 0.324855350985344 },
+        { 6430, 0.314630666062295, 0.324700368762729 },
+        { 6440, 0.314470994874025, 0.324545798213387 },
+        { 6450, 0.314311898787797, 0.324391638155726 },
+        { 6460, 0.314153375057231, 0.324237887409465 },
+        { 6470, 0.31399542095165, 0.324084544795669 },
+        { 6480, 0.313838033755984, 0.323931609136792 },
+        { 6490, 0.313681210770667, 0.323779079256713 },
+        { 6500, 0.313524949311538, 0.323626953980771 },
+        { 6510, 0.313369246709745, 0.323475232135805 },
+        { 6520, 0.313214100311644, 0.323323912550187 },
+        { 6530, 0.313059507478705, 0.323172994053855 },
+        { 6540, 0.312905465587415, 0.323022475478352 },
+        //@formatter:on
+    };
+
+    /**
+     * Search the 'KELVIN_TO_XY_LOOKUP_TABLE' for the XY entry closest to the given colour temperature.
+     * Uses a recursive 'QuickSearch' algorithm.
+     * 
+     * @param kelvin the colour temperature in K to find
+     * @param min the first index in the lookup table
+     * @param max the last index in the lookup table
+     * @return an array with the found CIE colour XY values
+     * @throws IndexOutOfBoundsException
```

Comment:
description is missing, causes a warning

Reply:
fixed